assignment/week2/velog/main.js

```diff
@@ -0,0 +1,67 @@
+const dateList = document.querySelector("#select-date-list");
+const dateSelectButton = document.querySelector("#select-default-date");
+
+const selecting = document.querySelectorAll(".selecting");
+const defaultDate = document.querySelector("#select-default-date-text");
+
+function showDateList() {
+  let dateListStatus = dateList.style.display;
+  if (dateListStatus === "block") {
+    document.querySelector("#select-date-list").classList.remove("add-block");
+  } else {
+    document.querySelector("#select-date-list").classList.add("add-block");
+  }
```

Comment:
```suggestion
    document.querySelector("#select-date-list").classList.toggle("add-block");
```

toggle메서드는 해당 클래스가 있으면 없애주고, 없으면 붙여주는 메서드인데 이렇게 클릭시 없어지고 나타나는 모달 구현할때 유용해요 나중에 참고하세여~~~

Reply:
헉 토글쓰니까 완전 간결해지는....최고...고마워요!!!